RunRoot/LoadIguana.C

```diff
@@ -0,0 +1,28 @@
+//setenv FMT .....(/usr/local?)
+//setenv IGUANA ....
+//setenv ROOT_INCLUDE_PATH ${FMT}/include:${ROOT_INCLUDE_PATH}
+
+void LoadIguana(){
+
+  //external iguana linking
+  TString IGUANA = gSystem->Getenv("IGUANA");
+  gInterpreter->AddIncludePath(IGUANA+"/include");
+  
+  gROOT->ProcessLine("#include <iguana/algorithms/AlgorithmBoilerplate.h>");
+  gROOT->ProcessLine("#include <iguana/algorithms/AlgorithmSequence.h>");
+  gROOT->ProcessLine("#include <iguana/algorithms/Algorithm.h>");
+ 
+  gSystem->Load("$IGUANA/lib/libIguanaServices.so");
+  gSystem->Load("$IGUANA/lib/libIguanaAlgorithms.so");
+  gSystem->Load("$IGUANA/lib/libIguanaValidators.so");
```

Comment:
If this is all `$IGUANA` is used for, then I don't think it's necessary, since `$ROOT_INCLUDE_PATH` and `$(DY)LD_LIBRARY_PATH` should be sufficient. Have you tested that?

Reply:
You are correct, in principle includes and lib directories can be linked in that way. However I do have a preference for the explicit setting of the variable for a couple fo reasons. I think it is simpler and clearer for inexperienced users installing on their own machines and when there are multiple versions of dependencies on a machine (in this case iguana) you can easily switch between them and know exactly which one you are using. For example on ifarm at the moment when I was trying to just use ROOT_INCLUDE_PATH for linking clas12root there turned out  to be an issue that module unload did not remove clas12root from the ROOT_INCLUDE_PATH and I got strange and difficult to debug errors.

I do not have particularly strong opinion on this, so could be persuaded to change.
Cheers
Derek